README.md

```diff
@@ -21,44 +21,51 @@ We intend to support:
 - and many more.
 
 ## Installation
-### from a cloned repository
 
-just run
+You can install this library in two different ways. 
+
+### Install via NPM/Yarn 
+
+```shell
+$ yarn add api-flow
+# or 
+$ npm install api-flow
+```
+
+### Install from source
+
+Just run:
 
 ```sh
 git clone https://github.com/luckymarmot/API-Flow.git
 cd API-Flow
 make install
 ```
 
-This will install the node module dependencies
-
-## Building the different libraries
-### node, web, and webworker
-
-run the following command to build API-Flow for the different environments that you need
+This will install the node module dependencies, but you will need to build API-Flow for the different environments that you need:
 
 ```sh
 # use TARGET="node" if you only want the node library
 make runners TARGET="node web webworker"
 ```
 
-### Paw
```

Comment:
It doesn't really explain how this is Paw related, kinda seems like this is for building outputs in general. Remember, a lot of people will have no idea that this repo has anything to do with paw. It's just an A <> B converter.